docs/guides/developer-tips/explorer-debugging.mdx

```diff
@@ -0,0 +1,61 @@
+# Explorer debugging
+
+Visit the Explorer at [https://explorer.hyperlane.xyz](https://explorer.hyperlane.xyz/)
+
+Then paste a sender/recipient address or a transaction hash into the top search field. All matching messages will be shown in the result list. Click the row for more details.
+
+<figure>
+![](../../../static/img/explorer-debugging/explorer-search.png)
+<figcaption>Retrieving a transaction in the Hyperlane Explorer.</figcaption>
+</figure>
+
+## Debugging failed messages
+
+You'll know that a message failed to process because the bar at the top of the Message Explorer will turn red and the upper right section of the page will feature error reasons indicating why the message wasn't able to process.
```

Comment:
This is no longer true, the message page was redesigned a while ago. Can you change to:

`The message status near the top of the page will show as "Error" and the upper right section of the page will specify a reason why the message wasn't able to be processed`

Reply:
Ah yes thanks! made a mental note to remove but forgot to do it